libs/elements/layout/carousel/src/lib/project-carousel/next.directive.ts

```diff
@@ -0,0 +1,16 @@
+import { Directive, ElementRef, HostListener } from '@angular/core';
+
+@Directive({
+  selector: '[elewaWebsiteNext]',
```

Comment:
Move both directives inside a directives folder . `lib/directives/`.

Reply:
do i have to create a new directive?